plugins/file-open-save-new/core/src/main/javascript/app/services/providers/vfs.service.js

```diff
@@ -241,7 +241,8 @@ define(
         function save(filename, folder, currentFilename, override) {
           select(JSON.stringify({
             name: filename,
-            parent: folder.path,
+            path: _getFilePath(folder),
```

Comment:
did you make a change to this section. It seems like this change was already merged. Do you need to rebase?

Reply:
A similar change was made to the `open` function but not the `save` function